policy-controller/src/main.rs

```diff
@@ -10,10 +10,17 @@ use std::net::SocketAddr;
 use structopt::StructOpt;
 use tokio::{sync::watch, time};
 use tracing::{debug, info, info_span, instrument, Instrument};
+use tracing_subscriber::{fmt::format, prelude::*, reload, EnvFilter};
 
 #[derive(Debug, StructOpt)]
 #[structopt(name = "policy", about = "A policy resource prototype")]
 struct Args {
+    #[structopt(parse(from_str = LogLevel::new), long, default_value = "linkerd=info,warn")]
+    log_level: LogLevel,
```

Comment:
also, i think we _could_ just use `EnvFilter`'s `FromStr` impl directly without having to go through the `LogLevel` type?
```suggestion
    #[structopt(parse(try_from_str), long, default_value = "linkerd=info,warn")]
    log_level: EnvFilter,
```